packages/p2p/src/components/buy-sell/buy-sell-table.jsx

```diff
@@ -19,7 +17,7 @@ export const BuySellTable = ({ setSelectedAd, table_type }) => {
                     <Table.Head>{localize('Limits')}</Table.Head>
                     <Table.Head>{localize('Price')}</Table.Head>
                     <Table.Head>{localize('Payment method')}</Table.Head>
-                    {!is_agent ? <Table.Head>{localize('Trade')}</Table.Head> : null}
+                    <Table.Head>{''}</Table.Head>
```

Comment:
```suggestion
                    <Table.Head />
```

Reply:
I want it to be explicit that it's an empty header with an empty column name on purpose, I think it's more important to be explicit in this case, what do you think?

Reply:
understood, i think its ok 👍 if thats the case